.github/workflows/main.yml

```diff
@@ -1,77 +1,272 @@
 name: CI
 
-on:
+on: 
   # push: -- just run on PRs for now
   pull_request:
   workflow_dispatch:
 
 jobs:
-  build:
+  build_target_branch:
     runs-on: ubuntu-latest
+
     services:
       postgres:
         image: postgis/postgis:13-3.0
         env:
           POSTGRES_PASSWORD: postgis
-          POSTGRES_DB: arches
+          POSTGRES_DB: arches_for_science
         ports:
           - 5432:5432
         options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
 
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.12"]
```

Comment:
Since Arches for Science is conceived primarily as a library, I guess I'd prefer to see the full range of python versions it supports in the test matrix. (I don't know why the project settings template was changed in this respect, as it didn't go through code review.)

Reply:
never mind! this is now build target branch only, false alarm!